src/Platform.Handlers/src/Xamarin.Platform.Handlers/Core/IFrameworkElement.cs

```diff
@@ -1,12 +1,11 @@
-﻿using System;
-using Xamarin.Forms;
+﻿using Xamarin.Forms;
 
 namespace Xamarin.Platform
 {
 	public interface IFrameworkElement
 	{
 		bool IsEnabled { get; }
-		Color BackgroundColor { get; }
+		Brush2 Background { get; }
```

Comment:
`Brush2` must be temporary. Brush makes use of BindableProperties and is a BindableObject in Xamarin.Forms.
How are we going to manage these cases?

Reply:
My idea is use interfaces like `IBrush` and share the interface between Xamarin.Forms and .NET MAUI.